### ISSUE TITLE
PCB Designing

### ISSUE THREAD

**Issue by joyalraju** [joyalraju/Capstone-Project-Summer-2020#16]

**Comment by mikeshams** [comment]
1. What issues did you or others might face when creating the PCB layout?
2. How to resolve them?

**Comment by sruthykrishnan** [comment]
1. EasyEDA has the provision to directly convert schematics to PCB design. But while doing that I experienced some difficulty in finding some components with footprints. This is one of the major issue that I faced during PCB design. Second issue is with auto-routing. The auto routed connections will be sometimes incomplete. It took a lot of time for me to establish connection as I couldn't convert the schematic to PCB. Sometimes it shows error message and tells to connect everything manually. Third issue was with the gerber file. EasyEDA has no provision for the gerber view. So for that I had to download a third party gerber viewer. Another issue with EasyEDA is, it cannot display the exact 3D view of the components.

2. As our conversion from schematic to PCB was not that successful, I had to redraw the design from scratch.
    For effective routing, auto route each component separately, which is a time consuming task as well.
    In order to show the gerber view, I downloaded a third party gerber viewer, Gerbv.
    To see the exact 3D view of PCB, another design software like Kicad can be used.
